en/middleware/uxrce_dds.md

```diff
@@ -15,44 +15,44 @@ In particular it covers the options that are most important to PX4 users.
 
 ## Architecture
 
-The XRCE-DDS middleware consists of a client running on PX4 and an agent running on the companion computer, with bi-directional data exchange between them over a serial or UDP link.
+The uXRCE-DDS middleware consists of a client running on PX4 and an agent running on the companion computer, with bi-directional data exchange between them over a serial or UDP link.
 The agent acts as a proxy for the client, enabling it to publish and subscribe to topics in the global DDS data space.
 
-![Architecture XRCE-DDS with ROS 2](../../assets/middleware/xrce_dds/architecture_xrce-dds_ros2.svg)
+![Architecture uXRCE-DDS with ROS 2](../../assets/middleware/xrce_dds/architecture_xrce-dds_ros2.svg)
 
-In order for PX4 uORB topics to be shared on the DDS network you will need _XRCE-DDS client_ running on PX4, connected to the _XRCE-DDS agent_ running on the companion.
+In order for PX4 uORB topics to be shared on the DDS network you will need _uXRCE-DDS client_ running on PX4, connected to the _micro XRCE-DDS agent_ running on the companion.
 
-The PX4 [microdds-client](../modules/modules_system.md#microdds-client) publishes to/from a defined set of uORB topics to the global DDS data space.
+The PX4 [uxrce_dds_client](../modules/modules_system.md#uxrce_dds_client) publishes to/from a defined set of uORB topics to the global DDS data space.
 
-The [eProsima XRCE-DDS _agent_](https://github.com/eProsima/Micro-XRCE-DDS-Agent) runs on the companion computer and acts as a proxy for the client in the DDS/ROS 2 network.
+The [eProsima micro XRCE-DDS _agent_](https://github.com/eProsima/Micro-XRCE-DDS-Agent) runs on the companion computer and acts as a proxy for the client in the DDS/ROS 2 network.
```

Comment:
See https://github.com/PX4/PX4-user_guide/pull/2483/files#r1183180317 - one possible reason to add that proposed note is to clear up why we use a client that does not have the same name as the  eProsima agent.